mx.py

```diff
@@ -4005,7 +4005,7 @@ def __init__(self, jdk, jvmArgs, mainClass, toolJar, buildArgs=None):
         preexec_fn, creationflags = _get_new_progress_group_args()
         if _opts.verbose:
             log(' '.join(map(pipes.quote, args)))
-        p = subprocess.Popen(args, preexec_fn=preexec_fn, creationflags=creationflags, stdout=subprocess.PIPE)
+        p = subprocess.Popen(args, preexec_fn=preexec_fn, creationflags=creationflags, stdout=subprocess.PIPE) #pylint: disable=subprocess-popen-preexec-fn
```

Comment:
A warning against using `preexec_fn` because this can apparently lead to dead-locks. I don't really know how to fix it, and it's working any ways, so I've disabled the warning.